driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/layout/LayoutCheck.java

```diff
@@ -481,8 +479,9 @@ private static double generateDistanceImage(
         }
 
         try {
-            // write image to given file
-            ImageIO.write(distanceImage, "PNG", new File(resultFilename));
+            // Write image to given file
+            resultFilename.toFile().getParentFile().mkdirs();
+            ImageIO.write(distanceImage, "PNG", new File(resultFilename.toAbsolutePath().toString()));
```

Comment:
`resultFilename.toAbsolutePath().toFile()`?
https://docs.oracle.com/javase/8/docs/api/java/nio/file/Path.html#toFile--

Reply:
`toFile()` does not create missing folders, but I need them for `ImageIO.write()`.

Reply:
Err... but did you not create them with the preceeding line `resultFilename.toFile().getParentFile().mkdirs()`?

Reply:
Ah... I was confused... You mean
`ImageIO.write(distanceImage, "PNG", resultFilename.toAbsolutePath().toFile())`?